Cascade/Cascade-Bridging-Header.h

```diff
@@ -0,0 +1,5 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
+
+#import <PureLayout/PureLayout.h>
```

Comment:
for some reasons I didn't receive the report from Travis (which is weird, because it spams me on every change and on all the fails of course)